astropy/io/fits/convenience.py

```diff
@@ -443,7 +444,7 @@ def writeto(filename, data, header=None, output_verify='exception',
 
 def table_to_hdu(table):
     """
-    Convert an astropy.table.Table object to a FITS BinTableHDU
+    Convert an `~astropy.table.Table` object to a FITS BinTableHDU
```

Comment:
@SaraOgaz Normally I would agree, because it doesn't make sense to change unrelated functions. But there was already a change in this line (accidental?) so another change in the same line was just obvious.